### PR TITLE
Add deployed-contracts-prod.json

### PR DESCRIPTION
**Pull request by clothic**
```
sed -e ':again' -e N -e '$!b again' -e 's/"hardhat": {[^}]*},//g' deployed-contracts.json > deployed-contracts-prod.json
sed -i.bak -e ':again' -e N -e '$!b again' -e 's/"localhost": {[^}]*},//g' deployed-contracts-prod.json
sed -i.bak -e ':again' -e N -e '$!b again' -e 's/"optimism_localhost": {[^}]*},//g' deployed-contracts-prod.json
sed -i.bak -e ':again' -e N -e '$!b again' -e 's/"base_localhost": {[^}]*},//g' deployed-contracts-prod.json
sed -i.bak -e ':again' -e N -e '$!b again' -e 's/"arbitrum_localhost": {[^}]*},//g' deployed-contracts-prod.json
```

### CODE REVIEW

File: packages/contracts/deployed-contracts.json
Comment (hudeintres) [inline packages/contracts/deployed-contracts.json:1]: wait don't delete this one, we still need it

Comment (hudeintres) [reply packages/contracts/deployed-contracts.json:1]: getDb would use this right

File: packages/contracts/helpers/contracts-helpers.ts [hunk 1]
```diff
@@ -39,27 +45,29 @@ import { getDefenderRelaySigner, usingDefender } from "./defender-utils";
...
-    // console.log(`tx: ${contractInstance.deployTransaction.hash}`);
-    // console.log(`contract address: ${contractInstance.address}`);
-    // console.log(`deployer address: ${contractInstance.deployTransaction.from}`);
-    // console.log(`gas price: ${contractInstance.deployTransaction.gasPrice}`);
-    // console.log(`gas used: ${contractInstance.deployTransaction.gasLimit}`);
-    // console.log(`\n******`);
-    // console.log();
-  }
 
+  if (isProdNetwork(currentNetwork)) {
```
Comment (hudeintres) [inline packages/contracts/helpers/contracts-helpers.ts:63]: so is the workflow to write separately to prod db and never have to copy from deployed-contracts again? If so, there's a lot of other places where we write to regular db. Like all the current deployment scripts under tasks will write to the regular db

File: packages/contracts/deployed-contracts.json
Comment (hudeintres) [reply packages/contracts/deployed-contracts.json:1]: also where would hardhat and localhost deployments go?

Comment (hudeintres) [reply packages/contracts/deployed-contracts.json:1]: or is this gitignored so you deleted it here?

Comment (clothic) [reply packages/contracts/deployed-contracts.json:1]: this is gitignored. local dev will have it

File: packages/contracts/helpers/contracts-helpers.ts (hunk 1 above)
Comment (clothic) [reply packages/contracts/helpers/contracts-helpers.ts:63]: yes, we don't have to manually copy. 

All deployment scripts will write to `deployed-contracts.json`. Only prod networks will write to the prod json. I don't understand what the issue is.

File: packages/contracts/deployed-contracts.json
Comment (clothic) [reply packages/contracts/deployed-contracts.json:1]: hardhat and localhost go into deployed-contracts.json

**Comment by clothic** [comment]
Approved offline

**Pull request merged**